src/assessment.js

```diff
@@ -49,5 +49,5 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
If you use `bind` here, you need to explicitly invoke it in order for your code to work.